game/static/game/css/leaderboard.css

```
@@ -0,0 +1,309 @@
+/* ============================================================
+   leaderboard.css — Styles for the Checkora leaderboard page
+   ============================================================ */
+
+body {
+    background: var(--bg-primary, #0f0f1a);
+    color: var(--text-primary, white);
+    font-family: 'Inter', Arial, sans-serif;
```

Stylelint (17.13.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)